install/index.php

```diff
@@ -1842,13 +1842,7 @@ function getSiteDir($cmsPath, $str) {
 function errorDisplayPage($errorTitle, $errorMsg, $showRefer = TRUE) {
   if ($showRefer) {
     $docLink = CRM_Utils_System::docURL2('Installation and Upgrades', FALSE, 'Installation Guide', NULL, NULL, "wiki");
-
-    if (function_exists('ts')) {
-      $errorMsg .= '<p>' . ts("<a %1>Refer to the online documentation for more information</a>", array(1 => "href='$docLink'")) . '</p>';
-    }
-    else {
-      $errorMsg .= '<p>' . sprintf("<a %s>Refer to the online documentation for more information</a>", "href='$docLink'") . '</p>';
-    }
+    $errorMsg .= '<p>' . ts("Refer to the online documentation for more information: ") . $docLink . '</p>';
```

Comment:
@wmortada Do you have a specific reason for removing the `function_exists('ts')` guard -- or just a general sense that it feels redundant?

Without a more specific reason, I'm inclined to keep the guard in there --the `install/index.php` is general a preboot environment where we can't take much for granted, and (as an error handler) it makes even more sense for `errorDisplayPage()` to be conscientious about preconditions.

Reply:
Ah, that's a good point.

CC @bgm -- any thoughts from i18n perspective? Are you persuaded too?